playbooks/ORCHESTRATION.md

```diff
@@ -127,9 +127,67 @@ Manually check for:
 
 <!-- TODO: Add playbook for disaster/recovery steps. -->
 
-### Hardware Key Signing
+## Step 3: Add delegation (Optional)
+
+This step will add a delegated role to the top-level targets that is
+controlled by an external GitHub repository. Coordinate with the
+delegation keyholder to run the `add-delegation` (see
+https://github.com/sigstore/root-signing/blob/main/cmd/tuf/app/add-delegation.go).
+After the delegation metadata is added and signed, the delegation
+keyholder should open a PR against the ceremony branch.
+The name of the PR MUST be `feat/add-delegation for
+<delegation-name>`. When creating the delegation, a  `target-meta`
+file has to be provided that lists the targets, similar when adding
+the top level targets.
+
+As part of running the `add-delegaton` command, a POP (proof of
+possession) has to be generated too. The computed POP should be stored
+in `${REPO}/staged/${FORK_POINT}.sig`, where the fork point is the
+fork point from `main` and the ceremony branch. This fork point is
+also used as the nonce when computing the pop.
```

Comment:
Can we note which command generates the PoP, just to make it clear it's handled as part of these steps?

Reply:
Yes, I'll updates.